tests/v2/validation/upgrade/README.md

```diff
@@ -7,12 +7,12 @@ In your config file, set the following, this will run each test in parallel both
 ```yaml
```

Comment:
I don't see any difference for the items in the array, the previous one makes it a bit more obvious IMO because the cluster name comes first, what do you think?

Reply:
I can revert it back, I think I originally modified it because there was some indentation issues, but I'll revert it back